cmd/dryrun/dryrun.go

```diff
@@ -72,6 +72,16 @@ func (m command) build() *cli.Command {
 				return fmt.Errorf("failed to read config file %s: %v", opts.envFile, err)
 			}
 
+			// if aws is the provider, set the region if provided
```

Comment:
Why are we duplicating this logic for each of the commands?

Reply:
good point let me check how can I unify